src/main/java/qupath/ext/instanseg/ui/InstanSegController.java

```diff
@@ -204,9 +207,15 @@ private InstanSegController(QuPathGUI qupath) throws IOException {
         configureThreadSpinner();
         configureInputChannelCombo();
         configureOutputChannelCombo();
+        configureOutputTypeCombo();
         configureDefaultValues();
     }
 
+    private void configureOutputTypeCombo() {
```

Comment:
This defaults to annotations, which will be wrong for all uses of InstanSeg for cell detection.